<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <item>
    <layout class="QHBoxLayout" name="cb_layout">
     <item>
      <widget class="QWidget" name="control_widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="9">
         <widget class="QPushButton" name="update_background_pushButton">
          <property name="text">
           <string>update background</string>
          </property>
         </widget>
        </item>
        <item row="1" column="9">
         <widget class="QCheckBox" name="background_subtract_checkBox">
          <property name="text">
           <string>background subtract</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QCheckBox" name="hw_connect_checkBox">
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>HW Connect</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QDoubleSpinBox" name="int_time_doubleSpinBox"/>
        </item>
        <item row="1" column="5">
         <widget class="QCheckBox" name="continuous_checkBox">
          <property name="text">
           <string>continuous</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QCheckBox" name="save_h5_checkBox">
          <property name="text">
           <string>Save h5</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5" colspan="2">
         <widget class="QPushButton" name="start_pushButton">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="plot_widget" native="true">
     <layout class="QVBoxLayout" name="plot_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
